roles/oneview_os_deployment_server/meta/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+galaxy_info:
+  role_name: oneview_os_deployment_server_facts
```

Comment:
role name should be oneview_os_deployment_server

Reply:
done